lib/services/auth.dart

```diff
@@ -118,9 +120,17 @@ class AuthenticationService {
     }
   }
 
-  Future<AuthResponse> _signInWithGoogle() async {
+  Future<AuthResponse?> _signInWithGoogle() async {
     _logger.info('Singing in with google');
 
+    if (kIsWeb) {
+      await _client.auth.signInWithOAuth(
+        OAuthProvider.google,
+        redirectTo: 'http://localhost:5000/',
```

Comment:
Maybe extract the link to constants?

Reply:
Yeah this is all a bit of a hack at the moment to get web working. We will need to pull this from the environment once we actually start supporting web

Reply:
I promise to make this betterer in follow ups